hack/common.sh

```diff
@@ -65,6 +67,39 @@ function go_build() {
     GOPROXY=off go build "$@"
 }
 
+function find_node_names() {
+    # OKD/OCP providers has different node names and does not have docker
+    if [[ $KUBEVIRT_PROVIDER =~ ocp.* ]]; then
```

Comment:
ocp and okd providers don't exist anymore. I think you don't have to keep code for them when moving things around.

Reply:
In general this can be condensed into two scenarios:

 * external cluster where we have no control
 * for all others, this can be simplified to something like `kubectl.sh get nodes -o name | sed "s#node/##g"`, removing special treatment for getting nodes.

Reply:
done